site/docs/model-audit/index.md

```diff
@@ -23,17 +36,9 @@ And displays the results:
 
 ## Purpose
 
-AI/ML models can introduce security risks through:
+AI/ML models can introduce significant security risks that traditional security tools miss. These threats include malicious code embedded in pickled models, suspicious TensorFlow operations, potentially unsafe Keras Lambda layers, encoded payloads hidden in model structures, risky configurations in model architectures, malicious content in ZIP archives, and embedded executables in binary model files.
```

Comment:
I kind of like the old version better.  Any reason why we compressed this to be less readable?